src/debug/unk_debug_menu_3.c

```diff
@@ -0,0 +1,362 @@
+#include "global.h"
+#include "debug.h"
+#include "constants/species.h"
+#include "constants/items.h"
+#include "string_util.h"
+#include "new_game.h"
+#include "load_save.h"
+#include "start_menu.h"
+#include "mystery_event_script.h"
+#include "berry.h"
+#include "mail_data.h"
+
+const u8 Str_842E238[] = _("タマゴ");
+
+#if DEBUG
+
+extern u8 gUnknown_Debug_845DAE1[];
+extern u8 gUnknown_Debug_845DAE1End[];
+
+size_t debug_sub_813C404(u8 * dest)
+{
+    size_t size = gUnknown_Debug_845DAE1End - gUnknown_Debug_845DAE1;
+    memcpy(dest, gUnknown_Debug_845DAE1, size);
+    unref_sub_812620C(dest, gUnknown_Debug_845DAE1);
```

Comment:
Looks like there are a bunch of calls to `unref` subs.  Might as well remove that prefix or replace with `debug`.